admin/class-automation.php

```diff
@@ -43,6 +43,46 @@ class Automation {
 			'location' => 'article_metadata',
 			'type'     => 'boolean',
 		],
+		'Section'     => [
+			'location' => 'apple_news_api',
+			'type'     => 'string',
+		],
+		'Slug'        => [
+			'location' => 'user_input',
+			'type'     => 'string',
+		],
+		'Theme'       => [
+			'location' => 'apple_new_api',
+			'type'     => 'string',
+		],
+	];
+
+	/**
+	 * The schema for automation rules.
+	 */
+	const SCHEMA = [
+		'type'  => 'array',
+		'items' => [
+			'type'       => 'object',
+			'properties' => [
+				'field'    => [
+					'default' => '',
+					'type'    => 'string',
+				],
+				'taxonomy' => [
+					'default' => '',
+					'type'    => 'string',
+				],
+				'term_id'  => [
+					'default' => 0,
+					'type'    => 'integer',
+				],
+				'value'    => [
+					'default' => 'false',
```

Comment:
Should be empty string:
```suggestion
					'default' => '',
```